package.json

```
@@ -13,7 +13,7 @@
     "lint"
   ],
   "engines": {
-    "node": "^10.13.0 || >=12.0.0"
+    "node": ">=12.0.0"
```

If supporting ESM, the following seems better:

```suggestion
    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
```

See https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#how-can-i-move-my-commonjs-project-to-esm